ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js

```diff
@@ -142,7 +142,7 @@ const CancelSpeedupPopover = () => {
           marginTop={4}
         >
           <Box className="cancel-speedup-popover__edit-gas-button">
-            <EditGasFeeButton />
+            {appIsLoading && <EditGasFeeButton />}
```

Comment:
Not sure I follow what this is achieving?

Reply:
EditGasFeeButton displays old estimate,  this change hides the old estimate and wait for it to be updated.

Reply:
Sorry maybe I'm still confused but shouldn't it be `!appIsLoading`?

Reply:
Thanks a lot for bringing this up, I just fixed.